proxy/core/src/main/scala/io/cloudstate/proxy/EntityDiscoveryManager.scala

```diff
@@ -83,16 +82,21 @@ object EntityDiscoveryManager {
            maxInboundMessageSize = config.getBytes("max-inbound-message-size"),
            relayOutputBufferSize = config.getInt("relay-buffer-size"),
            gracefulTerminationTimeout = Timeout(config.getDuration("graceful-termination-timeout").toMillis.millis),
-           passivationTimeout = Timeout(config.getDuration("passivation-timeout").toMillis.millis),
            numberOfShards = config.getInt("number-of-shards"),
            proxyParallelism = config.getInt("proxy-parallelism"),
-           journalEnabled = config.getBoolean("journal-enabled"),
+           journalEnabled = config.getBoolean("eventsourced-entity.journal-enabled"),
            config = config)
     }
 
     final def getConfig(path: String): Config =
       config.getConfig(path)
 
+    final def eventSourcedPassivation(): Timeout =
```

Comment:
Rather than a method, I think it would be good to have this as a setting value, like before. We could have an eventsourced settings class, to group these (journal enabled and passivation timeout currently), rather than have them all at the top level.

Reply:
Done

Reply:
Thanks